index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+          <div class="bar1"> 9:41
+          <img src="/images/Group 4.png" alt="icons"></div>
+          <div class="bar2"> 
+            <h1>TRACY MIRANJA</h1>
+          <img src="/images/Icon - Menu.png" alt="togglenav">
+        </div>
```

Comment:
- I think it will be better to replace your `<div class="header">` to a `header.` tag, like this 👉🏽 `<header class="header">` because typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. 🤗 

- Your images are currently not being rendered on the browser. Can we fix it? 

Here is what i can see 👇🏽 

<img width="438" alt="Screenshot 2022-12-01 at 17 03 30" src="https://user-images.githubusercontent.com/99246359/205101217-d964680b-74bf-4afd-948c-340003a56fd8.png">

I think you can render them by just simply adding a dot in front of `/images` and it would work. Like this `<img src="./images/Group 4.png" alt="icons">` 

⚠️ Kindly apply this to all images in your portfolio.